internal/operator/backrest/backup.go

```diff
@@ -97,12 +97,25 @@ func Backrest(namespace string, clientset kubeapi.Interface, task *crv1.Pgtask)
 	if cluster.Spec.Standby && cmd == crv1.PgtaskBackrestStanzaCreate {
 		repoType = crv1.BackrestStorageTypePosix
 	}
+	selector := fmt.Sprintf("%s=%s,%s", config.LABEL_PG_CLUSTER, cluster.Name, config.LABEL_PGO_BACKREST_REPO)
+	options := metav1.ListOptions{
+		FieldSelector: fields.OneTermEqualSelector("status.phase", string(v1.PodRunning)).String(),
+		LabelSelector: selector,
+	}
 
+	pods, err := clientset.CoreV1().Pods(cluster.Namespace).List(ctx, options)
+	if err != nil {
+		log.Errorf("error getting pods from selector %s: %s", selector, err)
+		return
+	} else if len(pods.Items) != 1 {
+		log.Error("pods returned does not equal 1, it should")
```

Comment:
Please rewrite it to human readable message)

Reply:
done in https://github.com/percona/percona-postgresql-operator/pull/45/commits/0a99c65b1a6637330ccb6eddbf005fd642976506